### PR TITLE
Create logo image for TeX in tex-gen subdir

### PR DESCRIPTION
**Pull request by bpedersen2**
This will make a later zipping of this folder easier.

### CODE REVIEW

File: indico/modules/events/util.py [hunk 1]
```diff
@@ -415,11 +415,11 @@ def get_base_ical_parameters(user, detail, path, params=None):
             'user_logged': user is not None, 'request_urls': request_urls}
 
 
-def create_event_logo_tmp_file(event):
+def create_event_logo_tmp_file(event, dir=config.TEMP_DIR):
```
Comment (ThiefMaster) [inline indico/modules/events/util.py]: call it `path` to avoid shadowing the builtin `dir`

Comment (ThiefMaster) [inline indico/modules/events/util.py]: the config is not accessible at import time (i'm surprised this doesn't break!). Set it to `None` here and do `dir or config.TEMP_DIR` inside the function

Comment (bpedersen2) [reply indico/modules/events/util.py]: Done

Comment (bpedersen2) [reply indico/modules/events/util.py]: Done

Comment (bpedersen2) [reply indico/modules/events/util.py]: Most probably there are no calls without setting the  dir path in the call left.

Comment (ThiefMaster) [reply indico/modules/events/util.py]: But default arguments are evaluated at import time, so it should already fail without even calling the function (and in the tests it did).

**Pull request closed without merging**